### DISCUSSION TITLE
oicq版本2.3.1,设备文件默认生成，从8月24日起频繁被封，今天彻底无法登录

### DISCUSSION THREAD

**Discussion by taidixiong233** [takayama-lily/oicq#417] [General]
安卓协议登录：
[安全提醒]当前登录存在安全风险，请使用常用设备或通过手机号登录。
登录失败，错误码：237,[安全提醒]当前登录存在安全风险，请使用常用设备或通过手机号登录

[安全提醒]该帐号因使用非官方QQ客户端被暂时冻结QQ客户端登录，请点击下方按钮了解详情及处理。
登录失败，错误码：40,[安全提醒]该帐号因使用非官方QQ客户端被暂时冻结QQ客户端登录，请点击下方按钮了解详情及处理。

Mac协议登录 
[安全提醒]该帐号因使用非官方QQ客户端被暂时冻结QQ客户端登录，请点击下方按钮了解详情及处理。
登录失败，错误码：40,[安全提醒]该帐号因使用非官方QQ客户端被暂时冻结QQ客户端登录，请点击下方按钮了解详情及处理。

手表登录，apad, ipad： 
[安全提醒]当前登录存在安全风险，请使用常用设备或通过手机号登录。
登录失败，错误码：237,[安全提醒]当前登录存在安全风险，请使用常用设备或通过手机号登录

账号注册时间：19年10月
初次使用oicq时间：2022年1月
稳定从2022年2月到7月，8月开始小频率封号，24日开始大量封号
账号未开放设备锁，且oicq生成的设备已通过安全检验
账号从未发送过xml和json消息
oicq实例中Config值均为默认，协议除外

扫描登录显示网络环境复杂，机器人一直使用固定公网ip
现将整个项目copy下在本地运行也同样错误

**Comment by takayama-lily** [comment]
你是使用密码登录还是扫码登录？建议一律使用密码登录，不要再使用扫码。
